src/stream/src/executor/lookup/impl_.rs

```diff
@@ -414,7 +414,7 @@ impl<S: StateStore> LookupExecutor<S> {
                             &lookup_row,
                             ..,
                             false,
-                            PrefetchOptions::new_for_exhaust_iter(),
```

Comment:
Why disable prefetch here?

Reply:
🤔 Because the origin prefetch algorithm is not supported. In our system, since the time of iter a block is much smaller than loading it from S3, I think it will do nothing optimization in cloud environment unless we run it in remote storage based on NVMe.

Reply:
There are two cases here:
1. The range provided is exact, meaning that all data within the range needs to be iterated. (Correct me if I am wrong) In this case, given that the iterator implementation will only prefetch when the key range is across more than one block, I think there is no harm to turn on prefetch in the `PretchOptions`.
2. The range provided is not exact, meaning that only some data within the range needs to be iterated. In this case, I agree we can turn prefetch off since we need more hints.